src/scripts/OSFramework/OSUI/Event/DOMEvents/Listeners/AbstractListener.ts

```diff
@@ -41,11 +41,15 @@ namespace OSFramework.OSUI.Event.DOMEvents.Listeners {
 		 * @param {GlobalEnum.HTMLEvent} eventType
 		 * @memberof AbstractListener
 		 */
-		constructor(eventTarget: HTMLElement | Window, eventType: GlobalEnum.HTMLEvent) {
+		constructor(
+			eventTarget: HTMLElement | Document | Window,
+			eventType: GlobalEnum.HTMLEvent | GlobalEnum.CustomEvent,
+			isCustomEvent = false
+		) {
 			super();
 			this._eventTarget = eventTarget;
 			this._eventType = eventType;
-			this._eventName = GlobalEnum.HTMLEvent.Prefix + this._eventType;
+			this._eventName = isCustomEvent === false ? GlobalEnum.HTMLEvent.Prefix + this._eventType : this._eventType;
```

Comment:
@BenOsodrac we should comment this condition to make it clear to who's reading the code what is the purpose of this

Reply:
done!

Reply:
@joselrio typo where? 🤔

Reply:
Saw it ;)